tests/GraphBLAS-sharp.Tests/Backend/Algorithms/PageRank.fs

```diff
@@ -0,0 +1,118 @@
+﻿module GraphBLAS.FSharp.Tests.Backend.Algorithms.PageRank
+
+open Expecto
+open GraphBLAS.FSharp
+open GraphBLAS.FSharp.Tests
+open GraphBLAS.FSharp.Tests.Context
+open GraphBLAS.FSharp.Objects.ClVectorExtensions
+open GraphBLAS.FSharp.Objects
+
+let private alpha = 0.85f
+let private accuracy = 0.00001f
```

Comment:
Expecto have some constants like that. We use they in tests

Reply:
Lowest possible accuracy in expecto is 1e-06, which is too high for the algorithm and always causes tests to fail